packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -12,11 +13,27 @@ import 'package:flutter/widgets.dart';
 import 'colors.dart';
 import 'theme.dart';
 import 'typography.dart';
+import 'constants.dart';
 
+const Duration _kDialAnimateDuration = const Duration(milliseconds: 200);
+const double _kTwoPi = 2 * math.PI;
+const int _kHoursPerDay = 24;
+const int _kHoursPerPeriod = 12;
+const int _kMinutesPerHour = 60;
+
+enum DayPeriod {
+  am,
+  pm,
+}
+
+/// A value representing a time during the day
 class TimeOfDay {
```

Comment:
Would the Dart date format class work here? I assume that we're going to need to internationalize.

https://www.dartdocs.org/documentation/intl/0.9.1/intl/DateFormat-class.html